build.sbt

```diff
@@ -110,8 +114,41 @@ lazy val scala = project.in(file("faunadb-scala"))
           url("http://www.joda.org/joda-time/apidocs/index.html"))
     })
 
+lazy val javaDsl = project.in(file("faunadb-java-dsl"))
+  .settings(publishSettings: _*)
+  .settings(
+    name := "faunadb-java-dsl",
+    crossPaths := false,
+    autoScalaLibrary := false,
+    exportJars := true,
+    javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
+    testOptions += Tests.Argument(TestFrameworks.JUnit, "+q", "-v"),
+    apiURL := Some(url("http://faunadb.github.io/faunadb-jvm/faunadb-java-dsl/api/")),
+
+    (javacOptions in doc) := Seq("-source", "1.7",
+      "-link", "http://docs.oracle.com/javase/7/docs/api/",
+      "-link", s"http://google.github.io/guava/releases/$guavaVersion/api/docs/",
+      "-link", s"http://fasterxml.github.io/jackson-databind/javadoc/$jacksonDocVersion/",
+      "-link", s"http://faunadb.github.io/faunadb-jvm/$driverVersion/faunadb-common/api/",
+      "-link", s"http://dropwizard.github.io/metrics/$metricsVersion/apidocs/",
+      "-link", "http://www.joda.org/joda-time/apidocs/"
+    ),
+
+    testOptions += Tests.Argument(TestFrameworks.JUnit, "-q"),
+    libraryDependencies ++= Seq(
+      "com.fasterxml.jackson.datatype" % "jackson-datatype-guava" % jacksonVersion,
+      "joda-time" % "joda-time" % jodaTimeVersion,
+      "org.joda" % "joda-convert" % "1.8.1",
```

Comment:
This is only to make the Proguard stop complaining about some "missing" annotations used by joda-time. It's actually one of the dependencies (optional) of joda-time, but for some reason Proguard don't look for transitive dependencies. I'll instruct Proguard to ignore those warnings.